### ISSUE TITLE
conv2d calculation results are inconsistent with pytorch

### ISSUE THREAD

**Issue by lvjunsetup** [tensorflow/tensorflow#37621]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information** 
- Have I written custom code (as opposed to using a stock
example script provided in TensorFlow): 
- OS Platform and Distribution (e.g.,
Linux Ubuntu 16.04): 
- TensorFlow installed from (source or
binary): - TensorFlow version (use command below): 
- Python version: 3.6.5
- CUDA/cuDNN version: - GPU model and memory:10.1

**Describe the current behavior**
For some specific data, the calculation results of tensorflow and pytorch are inconsistent
**Describe the expected behavior**
The calculation results of tensorflow and pytorch are consistent or the difference is small
**Standalone code to reproduce the issue** 

        if target_interface == 'conv1':
            weights_torch = torch.from_numpy(np.full((11, 11, 3, 1), 1, np.float64).transpose((3, 2, 0, 1)))
            output_pytorch_cpu = torch.from_numpy(
                F.conv2d(torch.from_numpy(input_pytorch.numpy().transpose((0, 3, 1, 2))), weights_torch, padding=0,
                         stride=4).numpy().transpose((0, 2, 3, 1)))
**Other info / logs** 
tensorflow_output:
31.42857361,31.4285183,31.42857361
69.24489594,69.24489594,59.3673439
100.89792633,100.89792633,100.89792633
133.26530457,133.26530457,133.26530457
60.22449875,60.22449875,60.22449875
187.46939087,187.46939087,187.46939087
205.918396,205.918396,205.918396
96.42860413,96.42860413,96.42860413
61.42854309,61.42854309,61.42854309
148.89811707,148.89811707,148.89811707
96.28572083,96.28572083,96.28572083
83.02045441,83.02045441,83.02045441
87.89794922,87.89794922,87.89794922
135.12240601,135.12240601,135.12240601

pytorch_output
31.428574,31.428518,31.428574
59.367344,69.244896,69.244896
100.89793,100.89793,100.89793
133.2653,133.2653,133.2653
60.2245,60.2245,60.2245
187.46939,187.46939,187.46939
205.9184,205.9184,205.9184
96.428604,96.428604,96.428604
61.428543,61.428543,61.428543
148.89812,148.89812,148.89812
96.28572,96.28572,96.28572
83.020454,83.020454,83.020454
87.89795,87.89795,87.89795
135.1224,135.1224,135.1224

**Comment by Saduf2019** [comment]
@lvjunsetup 
could you please share the tensorflow version on which the issue was faced for us to replicate this on our local.

**Comment by HopefulRational** [comment]
Hi @lvjunsetup . Could you please share relevant tensorflow code to reproduce the issue.
Thanks.

**Comment by Saduf2019** [comment]
@lvjunsetup
please update on the above comment

**Comment by lvjunsetup** [comment]
> @lvjunsetup
> could you please share the tensorflow version on which the issue was faced for us to replicate this on our local.

Sorry to reply to you so late. The version of tensorflow is 1.90. The error operators are conv2d () and avg_pooling (). Thank you.

**Comment by lvjunsetup** [comment]
> Hi @lvjunsetup . Could you please share relevant tensorflow code to reproduce the issue.
> Thanks.

This error is a fact, and it can be reproduced by using a large number of random tensors as input to conv2d () or avg_pooling (). Thank you.

**Comment by Saduf2019** [comment]
@lvjunsetup
 could you please provide with simple stand standalone indented code with all dependencies for us to replicate the issue in our local, this would help us resolve the issue faster

**Comment by Saduf2019** [comment]
@lvjunsetup
could you please update as per above comment

**Comment by Saduf2019** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
